src/main/groovy/edu/oregonstate/mist/locations/mapper/LocationMapper.groovy

```diff
@@ -183,4 +186,24 @@ class LocationMapper  {
             )
         }
     }
+
+    /**
+     *  Create a Geometry object for Attributes
+     * @param coordinates
+     * @return
+     */
+    private static Geometry getGeometry(Double[][][] coordinates) {
+        if (coordinates) {
+            String type
+            if (coordinates.length == 1) {
```

Comment:
Since the type parameter is calculated based on coordinates, how about creating a getter in the Geometry class that performs this calculation?